### ISSUE TITLE
Add historical stadium attendance

### ISSUE THREAD

**Issue by jkeena88** [Spartan97/JCS#8]
Or capacity if attendance can't be found. I guess if we have both attendance and capacity, or if I'll need to mix and match, we could have a column for both. You said ESPN may have attendance for the last few years. I'll try to find a good source for capacity/older info.

**Comment by Spartan97** [comment]
Yeah that might be the best bet for attendance. Just checked and sports-ref doesn't have it in their boxscores.

I assume you want per-game attendance, not annual numbers?

**Comment by jkeena88** [comment]
Yes, per-game attendance if we can get it.

**Comment by Spartan97** [comment]
Interesting note - Wikipeida has exact attendances for each games, at least for Notre Dame in 1978.

**Comment by jkeena88** [comment]
Every school back to 2003 (but on an annual basis):
http://www.ncaa.org/championships/statistics/ncaa-football-attendance
